include/vcpkg/remove.h

```diff
@@ -16,7 +16,10 @@ namespace vcpkg::Remove
         YES
     };
 
-    DECLARE_MESSAGE(RemovingPackage, (msg::spec), "", "Removing {spec}");
+    DECLARE_MESSAGE(RemovingPackage,
+                    (msg::action_index, msg::action_count, msg::spec),
+                    "",
+                    "Removing {action_index}/{action_count} {spec}");
```

Comment:
```suggestion
                    "[{action_index}/{action_count}] Removing {spec}");
```

I think this was dg0yt's suggestion? (same for other examples)

I don't have strong opinions here I just want to make sure you considered this.

Reply:
Imo [x/y] looks only good when every line is prefixed with [x/y]